manifests/mpm/disable_mpm_event.pp

```diff
@@ -0,0 +1,8 @@
+class apache::mpm::disable_mpm_event {
+  @exec { '/usr/sbin/a2dismod mpm_event':
```

Comment:
Why a virtual resource? Couldn't you just include the class only when it's needed? You can include multiple times

How about a name and the specify the command explicitly? This makes it easier if some distro decides to change it somehow.